benchmark/report/script/cli.py

```diff
@@ -0,0 +1,15 @@
+import argparse
+
+import report_processor
+
+INDEX_DIRECTORY = ".index"
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        epilog="""A program, that synchronizes files between two directories, using a configuration file."""
```

Comment:
Incorrect description.
```suggestion
        epilog="""K6 benchmark and reporting automation"""
```